.spelling

```diff
@@ -494,6 +494,7 @@ uncomment
 uncommented
 Undeploy
 unencrypted
+unix
```

Comment:
Please use proper capitalization in the text, Unix, and the spell checker should not complain
```suggestion
```